cli/medperf/entities/benchmark.py

```diff
@@ -22,42 +23,38 @@ class Benchmark(Entity):
     what models to run and how to evaluate them.
     """
 
-    def __init__(self, bmk_dict: dict):
+    description: Optional[str] = Field(None, max_length=20)
```

Comment:
The difference between how entities behave now and before, is that "forgetting" to pass an optional field wouldn't cause any error. Before this, the constructor would raise key errors if any field is missing. I am just concerned about silent errors. Let me know what do you think

If we are worried about how easy/difficult for us to create temp entities for tests, mocks in the tests folder were already facilitating this.